opendatahub-io/data-science-pipelines-operator
# fix: argo network policy

## The issue resolved by this Pull Request:
Resolves https://issues.redhat.com/browse/RHOAIENG-1711

## Testing instructions

* Deploy DSPO
* Deploy DSPA
* confirm you can: 
1. Start a run
2. run completes, the completion green icon shows up (confirms persistent agent can talk to apiserver) 
3. confirm a scheduled run is created and ran successfully (confirms scheduled workflow crd controller can talk to apiserver) 
4. confirm artifacts/experiments updates (confirms mlmd communication) 
5. confirm no errors in any of the pods
6. try to curl port 8443 and 8888 from outside the namespace in a different pod, 8443 should work, 8888 should not
